sklearn/inspection/_plot/decision_boundary.py

```diff
@@ -248,6 +254,13 @@ def from_estimator(
             For multiclass problems, :term:`predict` is selected when
             `response_method="auto"`.
 
+        class_label : int, float or str, default=None
+            When `response_method` return several columns (i.e. `"predict_proba"` and
+            `"decision_function"`), `class_label` specifies which column to use for
+            plotting.
```

Comment:
Maybe this would be more intuitive for the users? 

```suggestion
            When dealing with a multiclass problem, you can visualize one class against
            the other classes by specifying ``class_label``. This can be used in
            combination with ``"predict_proba"`` and ``"decision_function"`` passed
            ass ``response_method``.

            See this example for how this can be used: ...
```